python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -83,10 +83,12 @@ def _select_unique_ids_from_manifest_json(
 ) -> AbstractSet[str]:
     """Method to apply a selection string to an existing manifest.json file."""
     try:
+        import dbt.flags as flags
         import dbt.graph.cli as graph_cli
         import dbt.graph.selector as graph_selector
         from dbt.contracts.graph.manifest import Manifest
         from dbt.graph import SelectionSpec
+        from dbt.graph.selector_spec import IndirectSelection
```

Comment:
Have we tested to make sure that this won't raise the CheckError when using dbt >=1.0.0, but < 1.4

Reply:
the dbt_13X tox environment installs dbt 1.3, and runs all the same tests